src/Common/MoistThermodynamics/states.jl

```diff
@@ -75,8 +75,14 @@ struct PhaseEquil{FT} <: ThermodynamicState{FT}
   "temperature: computed via [`saturation_adjustment`](@ref)"
   T::FT
 end
-PhaseEquil(e_int::FT, ρ::FT, q_tot::FT) where {FT<:Real} =
-  PhaseEquil(e_int, ρ, q_tot, saturation_adjustment(e_int, ρ, q_tot))
+function PhaseEquil(e_int::FT,
+                    ρ::FT,
+                    q_tot::FT,
+                    tol::FT=FT(100),
+                    maxiter::Int=5,
```

Comment:
Can we have the default `maxiter = 1` here? I'd like to get the performance gains coming from that, if we can, without the users having to specify it. If the tests pass with a tolerance of 0.1 K, this will be good enough.